.travis.yml

```diff
@@ -31,7 +31,7 @@ after_success:
   - echo $TRAVIS_PULL_REQUEST
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then echo "ignore push build result for per submit"; exit 0; fi'
   - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then echo "pushing build result ..."; fi'
-  - \cp -rf ${DOCS_ROOT}/_site/* test-content
+  - mkdir test-content && \cp -rf ${DOCS_ROOT}/_site/* test-content
```

Comment:
@lamber-ken curious why this was needed now

Reply:
if `test-content` doesn't exits, `\cp` command doesn't work in macOS, so I think create it is better as you mentioned.

Reply:
it worked well during my previous tests in travis ci env.